src/staticswitcher/staticswitcher.c

```diff
@@ -130,7 +127,7 @@ isSwitchWin (CompWindow *w)
 
     if (!w->mapNum || w->attrib.map_state != IsViewable)
     {
-	if (staticswitcherGetMinimized (s))
+	if ( staticswitcherGetMinimized (s) && ( (!(staticswitcherGetDrawPopup (s)) && staticswitcherGetMinimizedWhenPopupHidden (s)) || (staticswitcherGetDrawPopup (s)) ) )
```

Comment:
Unnecessary parenthesis in (!(staticswitcherGetDrawPopup (s)) which could be !staticswitcherGetDrawPopup (s). Also this line is too long, it should be broken up so that no line is longer than 80 characters.